.env.example

```diff
@@ -2,6 +2,7 @@
 
 JWT_SECRET="super-seekret"
 CLAIM_TOOL_PATH="/path/to/binary/claim_tool"
+IPFS_BIN_PATH="/path/to/ipfs"
```

Comment:
Should this be `IPFS_CLIENT_PATH` to be consistent with `OTS_CLIENT_PATH`? The fact that it's a binary is just a detail right?

Reply:
Fixed in ea88617